### ISSUE TITLE
JSON to Server Error

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#539]
On 2/22/19 in live beta 8: Created event 200015 as "Complete". Then toggled it to "Incomplete" in edit details. Then went to put it back to "Complete" and got this toast message: Error. Event not updated. Error message: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data

On 4/2/19 in live beta 10: tried again to mark 200015 as complete. Get this toast message: Error. Event not updated. Error message: {"error":"Server Error (500)"}

**Comment by BlakeDraper** [comment]
@lprivette check into the swapping of complete/incomplete for event 200015

**Comment by lprivette** [comment]
I was unable to recreate this issue.

I tried creating a new event as complete, changing to incomplete, and then back to complete (also did the reverse?inverse? don't know which word fits better ha). Tested this with already created events as well.

Perhaps recent updates/fixes for the services have resolved this issue?

**Comment by BlakeDraper** [comment]
Yes, I believe this is a consequence of the 'live' Beta being behind most recent updates. We will need to promote the versions up the chain to the live Beta more frequently, but have not established a good process for this yet.

**Comment by JChipault** [comment]
Toggling to complete seems to work fine on most events but not on others. Currently, can't mark 200015, 200018, and 200034 as complete in live version. All give server error 500. Maybe moving updates live would fix it.

**Comment by JChipault** [comment]
Actually, just this morning I toggled event 200049 from complete to incomplete, made a couple of changes (updated numbers affected, added a species, added a location contact), then tried changing process status to "no action required" and marking event as complete but got the server error 500

**Comment by BlakeDraper** [comment]
@JChipault The next step will be to update the live with the latest version and then to see if this behavior continues. There has been enough time spent with the current changes on the test version that I think it should be fine to do. We are really just looking for any catastrophic issues with the version when it passes through the test stage.

**Comment by JChipault** [comment]
@BlakeDraper  v13 was pushed to the live site and I still get server error 500 when I try to mark these 4 events as complete
